client/printing_3d/print_3d_client.js

```diff
@@ -0,0 +1,20 @@
+const grpc = require('grpc');
+var messages = require('./print_3d_pb');
+var services = require('./print_3d_grpc_pb');
```

Comment:
Not an error but why would we use var instead of let?

Reply:
Good catch, we should actually be using `const` here. Do you think you could open an issue so we can change all of these imports in each client?